docs/docs/recipes/configure-storage-providers/README.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 10
+---
+
+# 💾 Configure storage providers
+
+To store user uploaded files, Logto uses a storage provider. Logto supports multiple storage providers, including AWS S3, Azure Storage. This recipe will show you how to configure a storage provider for Logto.
+
+The configuration is stored in DB's `logto_configs` table, but it is recommended to use the CLI to configure the storage provider.
```

Comment:
should be `systems`? since it's a global config but `logto_configs` is per-tenant

Reply:
updated, the core code need to update as well